Tutorials/Level1Tutorial.md

```diff
@@ -0,0 +1,793 @@
+verbose output of the graph construction and simulation by passing the `-v`
+or `--verbose=LEVEL` option.  The higher the `LEVEL` specified, the more verbose
+the output will be.  This is often useful for debugging new simulation configurations.  
+```
+$> sst -v /path/to/simulation.py
+$> sst --verbose=4 /path/to/simulation.py
+```
+
+* _Testing Simulation Graph Syntax_: One of the common issues that users encounter 
+when constructing complex simulation graphs is basic syntactic issues in their 
+respective input files that prevent SST from constructing a valid simulation.  It is 
+often useful to utilize the SST graph loader(s) to validate a graph using the 
+internal initialization functions without actually executing the simulation.  
+This can be performed using the following command.  Note that SST will report 
+that the simulation completed, but the reported simulation time will be 0s.
+```
+$> sst --run-mode=init /path/to/simulation.py
+Simulation is complete, simulated time: 0 s
+```
+
+* _Outputting DOT Configuration Graphs_: It is also often useful for debugging 
+or publication purposes to direct SST to output a GraphViz DOT file that contains 
+the configuration graph of the candidate simulation.  This can be performed 
+with or without actually executing the simulation.  We show examples for both 
+as follows:
+```
+$> sst --output-dot=foo.dot /path/to/simulation.py
+$> sst --run-mode=init --output-dot=foo.dot /path/to/simulation.dot
+$> dot -Tpdf foo.dot > foo.pdf
+```
+
+* _Configuring the DOT Verbostiy_: Similar to the previous example, SST also 
+provides a mechanism to control the level of detail included in the DOT graph 
+output.  Setting the `--dot-verbosity` option alonsgide the `--output-dot` option 
+will control this feature.  The verbosity option accepts an integer as an input. The 
+default level (0-1) only displays the component names and draws links between 
+components.  Setting the level to (2-3) will also display the component type.  Setting 
+the level to (4-5) will also display the component ports.  Levels (8-9) 
+will display the link name and latency.  Finally, any level greater than or 
+equal to 10 will display the MPI rank information for the partitioned graph.
+```
+$> sst --run-mode=init --output-dot=foo_small.dot --dot-verbosity=5 /path/to/simulation/py
+$> dot -Tpdf foo_small.dot > foo_small.pdf
+$> sst --run-mode=init --output-dot=foo_large.dot --dot-verbosity=10 /path/to/simulation/py
+$> dot -Tpdf foo_large.dot > foo_large.pdf
+```
+
+* _Converting Between Simulation File Types_: One of the latest features 
+provided by SST is the ability to develop simulation inputs using Python 
+or JSON syntax.  Python is often useful when developing large simulations 
+that require redundant components generated (multiple CPUs with 
+memory hierarchys).  However, JSON inputs are often 
+easier to discern the respective graph 
+connectivity given the inherent syntax of JSON.  SST provides an internal 
```

Comment:
should this be "inherent hierarchical syntax"?

Reply:
resolved in branch